packages/react-component-library/babel.config.js

```diff
@@ -45,7 +45,13 @@ module.exports = {
       ],
     },
     test: {
-      plugins: ['@babel/plugin-transform-modules-commonjs'],
+      plugins: [
+        'babel-plugin-styled-components',
+        '@babel/plugin-transform-modules-commonjs',
+      ],
+    },
+    'storybook-test': {
+      plugins: ['babel-plugin-styled-components'],
```

Comment:
Added a new BABEL_ENV with `babel-plugin-styled-components` to get the css prop working for `yarn storybook:test`. Also no longer seems to cause a performance regression having it in `test` so added there too for consistency.

Reply:
Is it worth breaking this change out into a seperate commit?